include/circt/Dialect/FIRRTL/FIRRTLOps.h

```diff
@@ -85,14 +86,12 @@ enum Flow { Source, Sink, Duplex };
 /// source is sink.  The reverse of sink is source.  The reverse of duplex is
 /// duplex.  The \p accumulatedFlow parameter sets the initial flow.  A user
 /// should normally \a not have to change this from its default of \p
-/// flow::Source.
-flow::Flow foldFlow(Value val, flow::Flow accumulatedFlow = flow::Source);
+/// Flow::Source.
+Flow foldFlow(Value val, Flow accumulatedFlow = Flow::Source);
 
-namespace kind {
-enum Kind { Port, Instance, Other };
-}
+enum class Kind { Port, Instance, Other };
 
-kind::Kind getDeclarationKind(Value val);
+Kind getDeclarationKind(Value val);
```

Comment:
Kind is a very generic name.  I'd recommend changing this to `DeclKind`.  Making it top level enum class makes sense!